debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/SourceInfoTest.java

```diff
@@ -122,16 +122,16 @@ public void assertSourceInfoContents(SourceInfo source,
         assertThat(source.hasOffset(REPLICA_SET_NAME)).isEqualTo(hasOffset);
 
         Map<String, ?> offset = source.lastOffset(REPLICA_SET_NAME);
-        assertThat(offset.get(SourceInfo.TIMESTAMP)).isEqualTo(timestamp.getTime());
-        assertThat(offset.get(SourceInfo.ORDER)).isEqualTo(timestamp.getInc());
+        assertThat(offset.get(SourceInfo.TIMESTAMP)).isEqualTo((timestamp != null) ? timestamp.getTime() : 0);
```

Comment:
Is it possible to introduce something like `io.debezium.connector.postgresql.DecoderDifferences` ?

Reply:
Likely yes -- at it will be useful for tests that include update descriptions. However I would rather do it as a follow up PR.